src/acuvity/response/helper.py

```diff
@@ -154,18 +160,18 @@ def _get_text_detections(
 
         if guard.name == GuardName.KEYWORD_DETECTOR:
             return self._get_text_detections_type(extraction.keywords, guard, TextualdetectionType.KEYWORD, extraction.detections, match_name)
-        elif guard.name == GuardName.SECRETS_DETECTOR:
+        if guard.name == GuardName.SECRETS_DETECTOR:
             return self._get_text_detections_type(extraction.secrets, guard, TextualdetectionType.SECRET, extraction.detections, match_name)
-        elif guard.name == GuardName.PII_DETECTOR:
+        if guard.name == GuardName.PII_DETECTOR:
             return self._get_text_detections_type(extraction.pi_is, guard, TextualdetectionType.PII, extraction.detections, match_name)
         return False, 0, 0
 
     def _get_text_detections_type(
         self,
-        lookup: Dict[str, float] | None,
+        lookup: Union[Dict[str, float] , None],
         guard: Guard,
         detection_type: TextualdetectionType,
-        detections: List[Textualdetection] | None,
+        detections: Union[List[Textualdetection], None],
```

Comment:
these are good changes because this would actually break on Python 3.9

When you test/develop against these SDKs, please use the minimum supported version when developing. This way you catch problems like this right away.